src/topology/algebra/monoid.lean

```diff
@@ -513,6 +514,16 @@ begin
   exact ⟨y, hi, hy⟩
 end
 
+@[to_additive] lemma prod_eventually_eq {X M : Type*} [comm_monoid M]
```

Comment:
To make the name closer to `eventually_eq.mul`:
```suggestion
@[to_additive] lemma eventually_eq_prod {X M : Type*} [comm_monoid M]
```

Can you move this lemma to line 503, before all the locally_finite/finprod stuff?